src/main/java/com/powsybl/openloadflow/network/AbstractLfNetworkLoaderPostProcessor.java

```diff
@@ -31,4 +31,9 @@ public void onBranchAdded(Object element, LfBranch lfBranch) {
     public void onInjectionAdded(Object element, LfBus lfBus) {
         // to implement
     }
+
+    @Override
+    public void onAreaAdded(Object element, LfArea lfArea) {
+        // to implement
```

Comment:
Forgotten TODO

Reply:
This comment is present in all the other methods of the class that are not implemented, I just copied without asking myself if this is rigth 😅

Reply:
My bad, I just misunderstood the comment. Thanks for the clarification!